app/uk/gov/hmrc/helptosavefrontend/views/register/create_account.scala.html

```diff
@@ -40,119 +42,159 @@
+            value = Value(content = Text(Messages("hts.reminder-confirmation.title.p1-1") +" "+
+                  PeriodUtils.getMessage(period)
+                  +" "+ Messages("hts.reminder-confirmation.title.p1-2"))),
+            actions = Some(Actions(
+            items = Seq(ActionItem(
+              href = routes.RegisterController.changeEmail.url,
+              content = Text(Messages("hts.register.create_account.change"))
+              ))
+            ))
+        ))))
       }
+    }
   </dl>
 
-  @summary_section_head(heading = Messages("hts.register.create_account.your-bank-details"))
+<h2 class="govuk-heading-m">@messages("hts.register.create_account.your-bank-details")</h2>
   <dl class="govuk-check-your-answers cya-questions-long">
-    @summary_row(Messages("hts.register.create_account.your-bank-details.sort-code"), display2CharFormat(bankDetails.sortCode.toString()),
-      summary_section_change(
-        (routes.RegisterController.changeBankDetails.url + "#sortCode"),
-        Messages("hts.register.create_account.change-your-sort-code"),
-        "change-your-sort-code"
-      )
-    )
-    @summary_row(Messages("hts.register.create_account.your-bank-details.account-number"), bankDetails.accountNumber,
-      summary_section_change(
-        (routes.RegisterController.changeBankDetails.url + "#accountNumber"),
-        Messages("hts.register.create_account.change-your-account-number"),
-        "change-your-account-number"
-      )
-    )
+      @viewHelpers.govukSummaryList(SummaryList(
+      rows = Seq(SummaryListRow(
+      key = Key(content = Text(Messages("hts.register.create_account.your-bank-details.sort-code"))),
+      value = Value(content = Text(Messages(display2CharFormat(bankDetails.sortCode.toString())))),
+      actions = Some(Actions(
+      items = Seq(ActionItem(
+      href = routes.RegisterController.changeBankDetails.url + "#sortCode",
+      content = Text(Messages("hts.register.create_account.change"))
+      ))
+      ))
+      ))))
+
+      @viewHelpers.govukSummaryList(SummaryList(
+      rows = Seq(SummaryListRow(
+      key = Key(content = Text(Messages("hts.register.create_account.your-bank-details.account-number"))),
+      value = Value(content = Text(Messages(bankDetails.accountNumber))),
+      actions = Some(Actions(
+      items = Seq(ActionItem(
+      href = routes.RegisterController.changeBankDetails.url + "#sortCode",
+      content = Text(Messages("hts.register.create_account.change"))
+      ))
+      ))
+      ))))
+
     @if(bankDetails.rollNumber.nonEmpty) {
-      @summary_row(Messages("hts.register.create_account.your-bank-details.roll-number"), bankDetails.rollNumber.getOrElse(""),
-        summary_section_change(
-          (routes.RegisterController.changeBankDetails.url + "#rollNumber"),
-          Messages("hts.register.create_account.change-your-roll-number"),
-          "change-your-roll-number"
-        )
-      )
+      @viewHelpers.govukSummaryList(SummaryList(
```

Comment:
this is not supposed to be a separate govukSummaryList but instead it should be a SummaryListRow inside the one and only summary list you already rendered above